isort/parse.py

```diff
@@ -38,17 +38,18 @@ def _infer_line_separator(contents: str) -> str:
     return "\n"
 
 
+_from_absolute_normalizer_pattern = re.compile(r"from[\t ]+(.+?)[\t ]+(c?import)[\t ]+")
+_from_relative_normalizer_pattern = re.compile(r"from[\t ]*(\.+[^\t ]*?)[\t ]*(c?import)[\t ]+")
```

Comment:
I would just move this before _infer_line_separator and call them with uppercase and starting without _ to be standard global variable. Eg. `FROM_ABSOLUTE_NORMALIZER_PATTERN = ...`.

Reply:
* I put these here because they're used by `normalize_line()` -- `_infer_line_separator` is not.
* I chose the names to reflect that these are for use in the module, not for importing and using outside of the module.

If these reasons aren't convincing, I can move these away from their usage site and rename them. :+1:

Reply:
@kurtmckee since those are not being imported in other files, it is ok to keep `_` in the beginning of the name, but I would at least keep them uppercase since we follow mostly Google Style Guide -> https://google.github.io/styleguide/pyguide.html#316-naming

It is a bit personal preference for organization, I always put all global variables before all methods and classes, but it does not hurt also to keep there.

Reply:
Nice, I appreciate the info re: Google Style Guide usage. Thanks!